src/assets/css/sections/benefit_section.css

```diff
@@ -0,0 +1,64 @@
+.benefit-section {
+  height: 543px;
```

Comment:
Why do we need set height here?

Reply:
I misunderstood when making a section should not set the height for it